Dockerfile

```diff
@@ -3,9 +3,9 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 #---------------------------------------------------------------------------------------------
 
-ARG PYTHON_VERSION="3.10.4"
+ARG PYTHON_VERSION="3.10"
 
-FROM python:${PYTHON_VERSION}-alpine3.15
+FROM python:${PYTHON_VERSION}-alpine
```

Comment:
Unpin the version of Python and Alpine Linux, so that we always use the latest and avoid the necessity of frequently bumping versions (https://github.com/Azure/azure-cli/pull/21079).

Reply:
Is there any cache mechanism when pull images?

Reply:
Locally yes. If you don't run `docker pull`, existing docker image will be used, but on ADO there is no cache, so it will always pull the latest.